scripts/variantstore/wdl/GvsExtractAvroFilesForHail.wdl

```diff
@@ -10,12 +10,12 @@ workflow GvsExtractAvroFilesForHail {
         String dataset_name
         String filter_set_name
         String call_set_identifier
-        Boolean use_classic_VQSR = true
+        Boolean use_VQSR_lite = true
         Int scatter_width = 10
     }
 
     String fq_gvs_dataset = "~{project_id}.~{dataset_name}"
-    String filter_set_info_tablename = if (use_classic_VQSR) then "filter_set_info" else "filter_set_info_vqsr_lite"
+    String filter_set_info_tablename = "filter_set_info"
```

Comment:
I think this was some vestigial testing code from a previous PR that got accidentally committed. That `_vqsr_lite` version of the table does not appear to be created anywhere under any circumstances, so once VQSR lite was made the default in this PR the Avro extract failed because it couldn't find the nonexistent table.